src/main/resources/openapi.json

```diff
@@ -2193,6 +2239,32 @@
                     }
                 ],
                 "nullable" : true
+            },
+            "CreateTopic": {
+                "title": "CreateTopic",
```

Comment:
I guess we should add the "required" field as well to have "topic_name" as required. It will help the OpenAPI validator to check it. It should allow to avoid having the double check on topic name in the code as well.